modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -0,0 +1,82 @@
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+compute: <2> <3>
+- architecture: ppc64le
+  hyperthreading: Enabled <4>
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane: <2> <3>
+  architecture: ppc64le
+  hyperthreading: Enabled <4>
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: rdr-test-ocp
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  powervs:
+    powervsResourceGroup: ""
+    region: eu-de
+    serviceInstanceID: ""
+    userID: 65b64c1f1c29460e8c2e4bbfbd893c2c
+    zone: eu-de-1
+vpcRegion : eu-de
+publish: External
+pullSecret: '{"auths": ...}' <1>
+sshKey: ssh-ed25519 AAAA... <1>
+----
+<1> Required. The installation program prompts you for this value.
+<2> If you do not provide these parameters and values, the installation program provides the default value.
+<3> The `controlPlane` section is a single mapping, but the compute section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Although both sections currently define a single machine pool, it is possible that future versions of {product-title} will support defining multiple compute pools during installation. Only one control plane pool is used.
+<4> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
+====
+//ifndef::openshift-origin[]
+//<5> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
+//+
+//[IMPORTANT]
+//====
+//The use of FIPS Validated or Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
```

Comment:
You can remove mention of FIPS Compliance from power vs doc (install-config.yaml) @alishaIBM

Reply:
Acknowledge.